src/utils/lenkeUtil.tsx

```diff
@@ -17,10 +18,13 @@ const lenkeTilModia = (personData: PersonData) => {
     personData.dialogmotekandidat;
   const skalTilOppfolgingsplanOversikt =
     personData.harOppfolgingsplanLPSBistandUbehandlet;
+  const skalTilAktivitetskrav = hasActiveAktivitetskravStatus(personData);
```

Comment:
Er en boolean, passer det med en is i stedenfor skal på disse skalTil* variablene? feks isForAktivitetskrav?